README.md

```diff
@@ -19,6 +20,8 @@ npm install --save git+https://github.com/StediInc/TypeSuite.git
 
 Or add it to your `package.json` manually.
 
+Don't miss the section on Current Limitations towards the bottom of this file.
```

Comment:
I think I would move this to the "currently in alpha" and link to it.

e.g. **Note: TypeSuite is currently in alpha and is subject to [several limitations](#current-limitations)...**

Reply:
Done.